source/ddbus/thin.d

```diff
@@ -376,7 +405,7 @@ struct DBusAny {
       TypeMismatchException if the DBus type of the current value of the
       DBusAny object is not the same as the DBus type used to represent T.
   +/
-  T get(T)() @property const
+  T get(T)() @property const 
```

Comment:
why the trailing space?

Reply:
it got there because of a dfmt bug, and apparently it got accidentally committed.